presenter/src/main/java/com/mashup/chinchin/presenter/main/MainActivity.kt

```diff
@@ -228,16 +230,54 @@ fun RecommendFriendsScreen(
     }
 }
 
+@Preview(showBackground = true)
+@Composable
+fun MoreScreenPreview() {
+    MoreScreen(texts = listOf("이용 안내", "만든이들", "피드백 보내기"))
+}
+
 @Composable
-fun MoreScreen() {
-    ConstraintLayout(modifier = Modifier.background(Color.Green)) {
-        val text = createRef()
-        Text("More", Modifier.constrainAs(text) {
-            top.linkTo(parent.top)
-            bottom.linkTo(parent.bottom)
-            start.linkTo(parent.start)
-            end.linkTo(parent.end)
-        })
+fun MoreScreen(texts: List<String>) {
+    Column(
+        modifier = Modifier
+            .padding(horizontal = 24.dp)
+            .fillMaxWidth()
+
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.logo),
+            contentDescription = "",
+            modifier = Modifier.padding(top = 17.dp, bottom = 10.dp),
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        LazyColumn {
+            items(texts) { text ->
+                Button(
+                    onClick = {},
+                    colors = ButtonDefaults.buttonColors(backgroundColor = MaterialTheme.colors.background),
+                    modifier = Modifier.defaultMinSize(minHeight = 1.dp, minWidth = 1.dp),
+                    elevation = ButtonDefaults.elevation(
+                        defaultElevation = 0.dp,
+                        pressedElevation = 0.dp,
+                    ),
+                    contentPadding = PaddingValues(vertical = 20.dp),
+                ) {
+                    Text(
+                        fontSize = 16.sp,
+                        color = Black,
+                        fontWeight = FontWeight.Bold,
+                        text = text,
+                    )
+                }
+                Divider(
+                    color = Gray_300,
+                    thickness = 0.5.dp,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                )
+            }
+        }
```

Comment:
내가 의도로 말했던 것은 아래 같은 부분이었어..!
누나가 해준거처럼 해도 되긴하는데 button click이랑 text를 따로 관리해야한다는 점이 있어서 suggest 된 부분 한 번 고민해봐줘도 좋을거같아
어프로브하겠습니당

```suggestion
    MoreItemButton(text = "이용 안내", onClickButton = {...})
    MoreItemButton(text = "만든이들", onClickButton = {...})
    MoreItemButton(text = "피드백 보내기", onClickButton = {...})
    
@Composable
fun MoreItemButton(
    text: String,
    onClickButton: () -> Unit,
) {
    Button(
        onClick = onClickButton(),
        colors = ButtonDefaults.buttonColors(backgroundColor = MaterialTheme.colors.background),
        modifier = Modifier.defaultMinSize(minHeight = 1.dp, minWidth = 1.dp),
        elevation = ButtonDefaults.elevation(
            defaultElevation = 0.dp,
            pressedElevation = 0.dp,
        ),
        contentPadding = PaddingValues(vertical = 20.dp),
    ) {
        Text(
            fontSize = 16.sp,
            color = Black,
            fontWeight = FontWeight.Bold,
            text = text,
        )
    }
    Divider(
        color = Gray_300,
        thickness = 0.5.dp,
        modifier = Modifier
            .fillMaxWidth()
    )
}
```

Reply:
우아~ 직접 코드까지 작성해주다니~ 완전 감동이양 !